champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/brute/data/BlockTossObject.java

```diff
@@ -329,6 +337,19 @@ public void impact(Player caster) {
         impactLocation.getWorld().playSound(impactLocation, Sound.ENTITY_ZOMBIE_BREAK_WOODEN_DOOR, 0.3f, 1.5f);
     }
 
+    private void doDamage(Location impactLocation, LivingEntity ent, Player caster, Boolean isDirect){
+        Vector knockback = ent.getLocation().toVector().subtract(impactLocation.toVector());
+        final double strength = (radius * radius - ent.getLocation().distanceSquared(impactLocation)) / (radius * radius);
+        VelocityData velocityData = new VelocityData(knockback, strength, false, 0.0, 0.0, 3.0, true);
+        UtilVelocity.velocity(ent, caster, velocityData);
+        if(isDirect) {
+            UtilDamage.doCustomDamage(new CustomDamageEvent(ent, caster, null, EntityDamageEvent.DamageCause.CUSTOM, damage, false, skill.getName()));
+        } else {
+            UtilDamage.doCustomDamage(new CustomDamageEvent(ent, caster, null, EntityDamageEvent.DamageCause.CUSTOM, damage / 2, false, skill.getName()));
+        }
+        UtilMessage.simpleMessage(ent, skill.getName(), "<alt2>%s</alt2> hit you with <alt>%s</alt>.", caster.getName(), skill.getName());
```

Comment:
If you have this here remove lines 332-336

Reply:
the previous damage line only prints for the user, not the person getting hit